hnn_core/network.py

```diff
@@ -398,14 +399,16 @@ class Spikes(object):
         Each gid corresponds to a type via Network::gid_ranges.
     vsoma : dict
         Dictionary indexed by gids containing somatic voltages
+    isoma : dict
+        Dictionary indexed by gids containing somatic currents
     times : numpy array
```

Comment:
In keeping with our previous convention to use built-in python objects when possible, could we change `times` to type list?

Reply:
This is super nitpicky so we can also save this for the next PR if we want.

Reply:
Eh this is tough. There is a lot of code the presumes `times` is an ndarray for logical indexing and other conveniences. There is even an explicit test to ensure that `times` is instantiated as an array and not a list in [`test_network.py`](https://github.com/jonescompneurolab/hnn-core/blob/b50f91dcf7c0cdeae87189c9a1dc97f56bddc0b4/hnn_core/tests/test_network.py#L49). 

The change would just involved recasting with `np.array()` in a variety of placed around the code. I'm fine with changing to a list in this PR but I wonder if `times` is worth making an exception...

Reply:
I believe `times` is an array in MNE. It is indeed quite convenient in many places.

Reply:
Let's leave it then.